.travis.yml

```diff
@@ -12,19 +12,26 @@ env:
     # PG_VERSION (docker) and PG_GIT_TAG (Git) should be the same
     - PG_VERSION=9.6.6 PG_GIT_TAG=REL9_6_6
     - PG_VERSION=10.2 PG_GIT_TAG=REL_10_2
+    - EXEC_BACKEND="-DEXEC_BACKEND=1" PG_VERSION=10 PG_GIT_TAG=REL_10_2
+    - EXEC_BACKEND="-DEXEC_BACKEND=1" PG_VERSION=9.6 PG_GIT_TAG=REL9_6_6
 before_install:
   # We need the PostgreSQL source for running the standard PostgreSQL
   # regression tests
   - git clone --branch ${PG_GIT_TAG} --depth 1 https://github.com/postgres/postgres.git /tmp/postgres
-  - docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres postgres:${PG_VERSION}-alpine
+  - if [[ -z "$EXEC_BACKEND" ]]; then docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres postgres:${PG_VERSION}-alpine; fi
+  - if [[ -n "$EXEC_BACKEND" ]]; then docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres jlockerman/postgres_exec_backend_images:alpine${PG_VERSION}; fi
```

Comment:
Remove `jlockerman`?

Reply:
Oh, just saw the convo you had with Mat and Rob about this. I think we should get the account set up before merging this PR, if it's not too difficult?

Reply:
It's running off an image in my docker-hub repo until we get timescale-ci set up.

Reply:
Race condition ^__^

Reply:
@JLockerman I have setup a `timescaledev` docker hub repo (it wouldn't allow names with hyphens :\ ) here: https://hub.docker.com/u/timescaledev/

If you request access I can grant it, or Slack your account name

Reply:
@RobAtticus sent you a my docker id on slack